### DISCUSSION TITLE
How to enable volume based discounts?

### DISCUSSION THREAD

**Discussion by northrn** [medusajs/medusa#7658] [Q&A]
I need to enable volume based pricing and minimum order quantities for my products: ie. you must buy 100 units of product X which starts at $10, and is reduced to $9 when you buy 150, etc.

From what I understand this is lacking from the admin UI (disappointing considering B2B is a major reason to use a headless setup), and the only way to achieve this is with the API. My question is where are we running these API requests to add the price rules to our products? This is unclear from what I have found on the topic.

**Comment by Sander-Voogt** [comment]
@northrn did you already found a solution for this ?
